Tangem/Modules/Send/StakingModel.swift

```diff
@@ -124,21 +129,31 @@ private extension StakingModel {
             .store(in: &bag)
     }
 
-    func inputDataDidChange(amount: Decimal, validator: String, approvePolicy: ApprovePolicy) {
+    func inputDataDidChange(amount: Decimal, validator: String, apr: Decimal?, approvePolicy: ApprovePolicy) {
```

Comment:
```suggestion
    func inputDataDidChange(amount: Decimal, validator: ValidatorInfo, approvePolicy: ApprovePolicy) {
```
мне кажется так лучше будет )

Reply:
убрал вообще после обсуждения в слаке